ledger/metrics/src/main/scala/com/daml/metrics/ParallelIndexerMetrics.scala

```diff
@@ -4,16 +4,22 @@
 package com.daml.metrics
 
 import com.daml.metrics.api.MetricDoc.MetricQualification.{Debug, Latency, Saturation, Traffic}
-import com.daml.metrics.api.MetricHandle.{Counter, MetricsFactory, Histogram, Timer}
+import com.daml.metrics.api.MetricHandle.{
+  Counter,
+  Histogram,
+  LabeledMetricsFactory,
+  MetricsFactory,
+  Timer,
+}
 import com.daml.metrics.api.dropwizard.DropwizardTimer
 import com.daml.metrics.api.{MetricDoc, MetricName}
 
-@MetricDoc.GroupTag(
-  representative = "daml.parallel_indexer.<stage>",
-  groupableClass = classOf[DatabaseMetrics],
-)
-class ParallelIndexerMetrics(prefix: MetricName, factory: MetricsFactory) {
-  val initialization = new DatabaseMetrics(prefix, "initialization", factory)
+class ParallelIndexerMetrics(
+    prefix: MetricName,
+    factory: MetricsFactory,
+    labeledMetricsFactory: LabeledMetricsFactory,
+) {
+  val initialization = new DatabaseMetrics(prefix :+ "initialization", labeledMetricsFactory)
 
   // Number of state updates persisted to the database
   // (after the effect of the corresponding Update is persisted into the database,
```

Comment:
We have a bunch of `executor` variables defined but unused below — can we remove them, please?

Reply:
One is for docs only, and one is actually used..not sure why IJ marks it as unused.